graphql/database/logs/logTypes.js

```diff
@@ -0,0 +1,24 @@
+import types from '../fieldTypes'
+
```

Comment:
The typing here isn't strictly necessary but is probably helpful as a guide to others consuming data and as one additional check on data soundness. I'm on board with it.

Reply:
Yeah I think the former was from my perspective as it serves as documentation in one spot for the differing log schemas.